cosmosdb/types.bal

```diff
@@ -354,12 +359,13 @@ public type ResourceReadOptions record {|
     string sessionToken?;
 |};
 
-# Represent the optional parameters which can be passed to the function when querying containers.
+# Optional parameters which can be passed to the function when querying containers.
 # 
-# + consistancyLevel - The consistency level override. Allowed values are `Strong`, `Bounded`, `Session` or `Eventual`.
-#                      The override must be the same or weaker than the account’s configured consistency level.
+# + consistancyLevel - The consistency level override
```

Comment:
Suggestion: "The consistency level override. Possible values are `Strong`, `Bounded`, `Session`, and `Eventual`. The override must be the same or weaker than the Cosmos DB account’s configured consistency level."